src/trade/tradeConf.js

```diff
@@ -240,7 +240,7 @@ define(['lodash', 'jquery', 'moment', 'websockets/binary_websockets', 'common/ri
               if(this.category === 'Asians') {
                 //https://trello.com/c/ticslmb4/518-incorrect-decimal-points-for-asian-average
                 var avg = this.average().toFixed(decimal_digits + 1);
-                return {value: avg, label:'Average ('.i18n() + avg + ')', id: 'plot-barrier-y'};
+                return {value: avg*1, label:'Average ('.i18n() + avg + ')', id: 'plot-barrier-y'};
```

Comment:
`.toFixed()` returns a string, the *1 converts it back into a number. If you want to make the behaviour clearer, you could use `parseFloat(avg)` instead.